.github/workflows/run_tests.yml

```diff
@@ -17,14 +17,17 @@ jobs:
         platform: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Should we add `3.11` to `python-version`?

Reply:
h5py doesn't have prebuild for 3.11 which leads python to compile it and cause error. I would recommend to wait h5py upload precompile binary to pypi.